stdlib/private/SwiftReflectionTest/SwiftReflectionTest.swift

```diff
@@ -82,12 +99,9 @@ internal struct ReflectionInfo : Sequence {
   }
 }
 
-#if arch(x86_64) || arch(arm64)
-typealias MachHeader = mach_header_64
-#else
-typealias MachHeader = mach_header
-#endif
 
+// macOS specific functions
+#if os(macOS)
```

Comment:
There are 5 functions that are OS specific: `getSectionInfo`, `getAddressInfoForImage(atIndex:)`, `sendImages` (which just fails on Linux), `getReflectionInfoForImage(atIndex:)`, and `getImageCount`. I grouped them in #if... #else blocks, but I could create a protocol and implement them in different files instead.

Reply:
I don't know that we need a protocol, your current approach seems fine.

One request: please make an effort to absolutely minimize the number of #if...#else blocks, as each such block creates some maintenance burden.